app/assets/javascripts/Components/test_script_result_table.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import {render} from 'react-dom';
+import ReactTable from 'react-table';
+import _ from 'lodash';
+
+class TestScriptResultTable extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      data: [],
+    };
+    this.fetchData = this.fetchData.bind(this);
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  fetchData() {
+    $.ajax({
+      url: Routes.automated_tests_path(),
+      dataType: 'json',
+    }).then(res => {
+      this.setState({data: res});
+    });
+  }
+
+  render() {
+    const {data} = this.state;
+    return(
+      <div>
+        <h2>beeppeppe</h2>
+        <ReactTable
+          data={data}
+          columns={[
+            {
+              Header: "Runs",
+              accessor: "test_run_id"
+            },
+            {
+              Header: "File Name",
```

Comment:
Make sure you use the internationalized strings for the column headers, using `I18n.t(...)`.